src/analysis/bounds.rs

```diff
@@ -56,6 +56,7 @@ pub struct Bound {
     pub alias: char,
     pub type_str: String,
     pub info_for_next_type: bool,
+    pub hack: bool,
```

Comment:
What does this mean? A comment would be useful, a `hack` flag is not really self-explanatory :)